build-system/tasks/runtime-test.js

```diff
@@ -28,22 +28,37 @@ var webserver = require('gulp-webserver');
 var app = require('../test-server').app;
 var karmaDefault = require('./karma.conf');
 
+
+/**
+ * Logs a message if tests are being run during local development.
+ */
+function logLocal() {
+  if (!process.env.TRAVIS) {
+    util.log(util.colors.green.apply(null, arguments));
+  }
+}
+
 /**
  * Read in and process the configuration settings for karma
  * @return {!Object} Karma configuration
  */
 function getConfig() {
   if (argv.safari) {
+    logLocal(util.colors.yellow('--safari:'), 'Running tests on Safari.');
```

Comment:
Minor: This function only creates a config object and doesn't necessarily run tests.

Instead of sprinkling log calls around, WDYT about having a map of `argv` keys to descriptive messages at the beginning of the `test` task?

```js
gulp.task('test', ...) {
  const argvMessages = {
    safari: 'Running tests on Safari.',
    // ...
  };
  if (!process.env.TRAVIS) {
    Object.keys(argv).forEach(arg => {
      util.log(util.colors.green('--' + arg), argvMessages[arg] || '');
    });
  }
}
```